exercises/transpose/transpose_test.go

```diff
@@ -0,0 +1,44 @@
+package transpose
+
+import (
+	"fmt"
+	"testing"
+)
+
+// For a more visual representation of the test data than is provided by
+// cases_test.go have a look at the original json data here:
+// https://github.com/exercism/x-common/blob/master/exercises/transpose/canonical-data.json
+
+func TestTranspose(t *testing.T) {
+	for _, test := range testCases {
+		actual := Transpose(test.input)
+		if fmt.Sprintf("%#v", actual) != fmt.Sprintf("%#v", test.expected) {
```

Comment:
I've changed to to `reflect.DeepEqual` anyway, just to avoid any unforeseen problems and as it may get changed to this in the future anyway.